src/plugins/controls/public/controls/timeslider_control/get_timeslider_control_factory.tsx

```diff
@@ -284,7 +283,9 @@ export const getTimesliderControlFactory = (): ControlFactory<
           const to = useMemo(() => {
             return timeslice ? timeslice[TO_INDEX] : timeRangeMeta.timeRangeMax;
           }, [timeslice, timeRangeMeta.timeRangeMax]);
-
+          const value: Timeslice = useMemo(() => {
+            return [from, to];
+          }, [from, to]);
```

Comment:
**nit** It's a bit cleaner to have some whitespace before the `return` so that it doesn't turn into a gigantic blur :)

```suggestion
          }, [from, to]);
          
```